packages/commonwealth/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "add-component-showcase": "npx ts-node -T ./scripts/add-component-showcase.ts",
-    "build": "tsc -b && tsc-alias",
+    "build": "tsc -b",
     "clean": "rm -rf build",
     "check-types": "tsc --noEmit",
```

Comment:
This would use the `packages/commonwealth/tsconfig.json` config so it would pass with the bad imports no? Likely need to temporarily have 2 `tsc --noEmit` checks here -> one for the client and one for the server.

Reply:
Yeah so this might be a problem. Basically we don't get any build errors from adding these imports, but we do get editor visual errors:
<img width="1006" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14794654/5f2b5430-2d15-40d9-aab3-f85f13780ff6">

The only issue with having a separate check-types is that tsc won't actually work because the folder that you are type checking are not allowed to have imports outside the root (In this case we have server importing modules from other packages like @hicommonwealth/core). So I am not sure the solution for this one. Maybe its better just to settle for the editor errors.

Reply:
@kurtisassad per convo we should definitely not settle for editor errors. I'm not exactly sure what the solution is here but maybe we should hop in huddle with @Rotorsoft figure this out this week.